lib/elsa/group/manager.ex

```diff
@@ -246,8 +272,40 @@ defmodule Elsa.Group.Manager do
     end)
   end
 
+  defp start_workers(state, generation_id, assignments) do
+    Enum.reduce(assignments, state.workers, fn assignment, workers ->
+      WorkerManager.start_worker(workers, generation_id, assignment, state)
+    end)
+  end
+
   defp wait_and_stop(reason, state) do
     Process.sleep(2_000)
     {:stop, reason, state}
   end
+
+  defp custom_ack_hack?(name) do
```

Comment:
This the name you wanna go with for the final implementation?

Reply:
I wasn't going to say anything. ;)